### ISSUE TITLE
Is this project (basically) dead?

### ISSUE THREAD

**Issue by szsoftware** [raamcosta/compose-destinations#664]
Hi there,

after very enthusiastic and long during development it's quite quiet here since a while.

@raamcosta, no pressure!

But it would be good to know if there is more activity expected in the near future or is it worth to fork and let some people develop there?

After the strong advertisement of ComposeDestinations v2 I already migrated some projects and refactored  a lot in the expectation of release state.

For some projects I think I stay in the v1 branch as long there is not so much activity here.

I have full understanding of your priorities, new job, family, etc.

It's just some curiosity.

I wish you all the best and good luck in the things you're doing!

**Comment by raamcosta** [comment]
Hi @szsoftware 

No, definitely not dead! Actually, the reason there are no commits on main for some time now is because the time I've had to work on the library, I've dedicated it to KMP support 🙂 

Because KMP support of the official library came at this weird timing with m v2 version, it threw a bit of wrench in my plans because now I'm wondering if I should make v2 support KMP. It kinda makes sense right? V2 hits and it also supports KMP!

Nevertheless, I was already planning some updates for this weekend, mostly to use new official navigation and kotlin versions. At the same time, I'm still playing with KMP support, which should come soon in some form! (I already have all samples of this repo running on iOS and desktop! 😋 ).

I'm closing this, I don't want an open issue with this title, as it sends the wrong message just by existing :)
